src/polymer/behavior-scanner.ts

```diff
@@ -67,11 +67,34 @@ class BehaviorVisitor implements Visitor {
   propertyHandlers: PropertyHandlers|null = null;
 
   document: JavaScriptDocument;
+
   constructor(document: JavaScriptDocument) {
     this.document = document;
   }
 
   /**
+   * Fix how comments are attached to SequenceExpressions. In espree, any
+   * leading comments to the expression are parsed as leading comments to the
+   * entire sequence and not the first expression in that sequence.
+   *
+   * For our behaviors we expect our `@polymerBehavior` comments to be the
+   * leading comments of the actual first expression, so we copy them over here
+   * before so that other enter functions can properly read them.
+   */
+  enterSequenceExpression(
+      node: estree.SequenceExpression, parent: estree.Node) {
+    node.expressions.forEach((expression, i) => {
+      if (i === 0 && !expression.leadingComments) {
+        node.leadingComments = node.leadingComments || parent.leadingComments;
```

Comment:
Consider extracting this out instead to a `getLeadingComments` function somewhere, maybe on JavascriptDocument. There are a lot of cases where we want to do this sort of thing, and I bet this isn't the only place where we could do some parental walking to properly associate the right comments on the right nodes.